LoopKitUI/InsulinKit/InsulinDeliveryTableViewController.swift

```diff
@@ -365,14 +381,15 @@ public final class InsulinDeliveryTableViewController: UITableViewController {
                 let time = timeFormatter.string(from: entry.startDate)
 
                 cell.textLabel?.text = "\(volume) U"
+                cell.textLabel?.textColor = .label
                 cell.detailTextLabel?.text = time
                 cell.accessoryType = .none
                 cell.selectionStyle = .none
             case .history(let values):
                 let entry = values[indexPath.row]
                 let time = timeFormatter.string(from: entry.date)
 
-                cell.textLabel?.text = entry.title ?? LocalizedString("Unknown", comment: "The default title to use when an entry has none")
+                cell.textLabel?.attributedText = entry.dose?.localizedAttributedDescription ?? NSAttributedString(string: LocalizedString("Unknown", comment: "The default description to use when an entry has no dose description"))
```

Comment:
[nit] What do you think about:
```
if let attributedText = entry.dose?.localizedAttributedDescription {
  cell.textLabel?.attributedText = attributedText
} else {
  cell.textLabel?.text = LocalizedString("Unknown", comment: "The default description to use when an entry has no dose description")
}
```
This seems more accurate and gets rid of the artificial `NSAttributedString(string:)` wrapper.